views/index.ejs

```diff
@@ -67,79 +68,128 @@
   </nav>
 
   <div class="row">
-    <div class="col-md-12 m-body">
-
+    </div>
+</div>
       <div>
         <!-- Insert html view templates here -->
           <ui-view></ui-view>
 
+     
+    </div>
+  
+
+<!--_________________________________________________________________________-->
+
+  <script type="text/ng-template" id="/home.html">
+  <link rel="stylesheet"  type="text/css" href="style2.css">
+  <link rel="stylesheet"  type="text/css" href="vertical.css">
+
+<div class="container-fluid">
+  
+        
+  <div class="row">
+    <div class="col-sm-2 "   style="background-color:#d2b481;position:fixed">
```

Comment:
Add a class here with a name & put this property into that class.
Add the class here